### ISSUE TITLE
Releases are very old

### ISSUE THREAD

**Issue by c0nd3v** [g-truc/gli#166]
Releases are 3 years old now, could we get an update? I couldn't even get the latest release to work. I had to use master branch

**Comment by noizex** [comment]
Yeah, could we please have a release? It's been so many years and so many GLM versions - it's really pain for any package managers maintainers to keep this up-to-date with code that only works from master branch and there is no proper release tag...
